modules/ROOT/pages/sdk-xattr-example.adoc

```diff
@@ -231,13 +231,12 @@ public class GetTTL
  
         test.touch(key, 0);
  
-        //Multiple paths can be access at once via subdoc. It's limited to 16 paths and xattrs have to be frist
+        //Multiple paths can be access at once via subdoc. It's limited to 16 paths and xattrs have to be first
         DocumentFragment<Lookup> mutlisubpath = test.lookupIn(key).get("$document.exptime", new SubdocOptionsBuilder().xattr(true))
                                                                   .get("$document.value_bytes", new SubdocOptionsBuilder().xattr(true))
                                                                   .get("click")
                                                                   .execute();
  
-        System.out.print("\n\n");
```

Comment:
I guess what I was referring to is not necessarily removing it, but if you want to pad with line breaks, it'd be more normal to add them on the end of the previous println.  I think I know where these came from.  It has a code smell like a shell script/C developer hacked it up.

Reply:
Well, it came from https://issues.couchbase.com/browse/JCBC-1187?focusedCommentId=266347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-266347
But I ran it past Michael, too.

Will try and put something better together for next week, but I'm also from the shell hacker background. ;)

Reply:
Can we at least fix the typo?  :)  emacs and a plethora of free tools can identify typos and run style checks.

mutlisubpath => mut li sub path?

What's a mut li?

It'd have been easier to spot with camelcase.  multiSubPath.